src/vs/workbench/common/theme.ts

```diff
@@ -288,15 +288,20 @@ export const ACTIVITY_BAR_FOREGROUND = registerColor('activityBar.foreground', {
 	dark: Color.white,
 	light: Color.white,
 	hc: Color.white
-}, nls.localize('activityBarForeground', "Activity bar foreground color (e.g. used for the icons). The activity bar is showing on the far left or right and allows to switch between views of the side bar."));
+}, nls.localize('activityBarForeground', "Activity bar item foreground color when it is active. The activity bar is showing on the far left or right and allows to switch between views of the side bar."));
+
+export const ACTIVITY_BAR_INACTIVE_FOREGROUND = registerColor('activityBar.inactiveForeground', {
```

Comment:
@sandy081 this will break theme authors that do not update their themes. to preserve the old behaviour the inactive color should be derived from the active color using 0.6 transparency and not WHITE.

Reply:
Good point, my intention was to not to break, hence I used the transparency but I should derived it from active foreground colour as you said. 👍

Reply:
@sandy081 easy, like this: https://github.com/Microsoft/vscode/blob/80a9557b4afe739fee4659f87f77718895c3b737/src/vs/workbench/common/theme.ts#L46